content/en/docs/setup/install/operator/index.md

```diff
@@ -224,48 +224,63 @@ $ helm install istio-operator manifests/charts/istio-operator \
   --set tag={{< istio_full_version >}} \
   --set operatorNamespace=istio-operator \
   --set watchedNamespaces=istio-system \
-  --set revision=1-7-0
+  --set revision=1-8-1
 {{< /text >}}
 
 Note that you need to [download the Istio release](/docs/setup/getting-started/#download)
 to run the above command.
 {{< /tip >}}
 
-After running the command, you will have two control plane deployments and services running side-by-side:
+Copy the `IstioOperator` CR named `example-istiocontrolplane` from the cluster as `example-istiocontrolplane-1-8-1.yaml`, update the `IstioOperator` CR name to `example-istiocontrolplane-1-8-1` and add `revision: 1-8-1` to `IstioOperator` CR, the updated `IstioOperator` CR should resemble the following snippet:
 
 {{< text bash >}}
-$ kubectl get pods -n istio-system -l app=istiod
-NAME                             READY   STATUS    RESTARTS   AGE
-istiod-5f4f9dd5fc-4xc8p          1/1     Running   0          10m
-istiod-1-8-1-55887f699c-t8bh8    1/1     Running   0          8m13s
+$ cat example-istiocontrolplane-1-8-1.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+  name: example-istiocontrolplane-1-8-1
+spec:
+  revision: 1-8-1
```

Comment:
Should also add `profile: default`?

Reply:
done

Reply:
default is the default if profile is unset - we should be consistent with what we have elsewhere, which is to omit it, otherwise it may confuse users to mix these two.